### DISCUSSION TITLE
Add Tranquillien Tabard to the reward catalogue

### DISCUSSION THREAD

**Discussion by FFabz** [chromiecraft/chromiecraft#4295] [Suggestions]
Tranquilian tabard is a tabard in the game since TBC but was never released. The model is in the game files, I was able to equip it on the PTR. Would be nice to have it in the reward catalogue for those who want to wear it as their is no way to get it otherwise.

https://www.wowhead.com/wotlk/item=23388/tranquillien-tabard

Edit: Two other tabards are in the game files but not available :
https://www.wowhead.com/wotlk/item=23710/upperdeck-tabard-3
https://www.wowhead.com/wotlk/item=11364/tabard-of-stormwind

thanks

**Comment by BeefSteaker** [comment]
Stormwind one is interesting as it's not faction locked, but the other two would definitely be nice to add

**Comment by Amrx2** [comment]
Hi Ffabz, thanks for this. Keep your eyes pealed, you might seen them in our shop one day.

Cheers,
Construct

**Reply by FFabz** [reply]
friendly reminder :)

**Comment by FFabz** [comment]
up
